packages/utils/global-nodes.ts

```diff
@@ -0,0 +1,34 @@
+import isServer from './isServer'
+
+const globalNodes = []
+let target = isServer ? void 0 : document.body
+
+export function createGlobalNode(id?: string) {
+  const el = document.createElement('div')
+
+  if (id !== void 0) {
```

Comment:
void 0 seems weird.

Reply:
Yeah, we should use `isUndefined` instead.